participants/CloudSpeed.md

```diff
@@ -0,0 +1,64 @@
+# CloudSpeed
+
+## (0) Project Name and Team Slack Handles
+
+*Our project Name:* **CloudSpeed**
+
+*Team Slack Handles:* **derek**
+
+## (1) Brief Project Description
+
+**CloudSpeed** *has built a sharing platform with creative works library as the core, providing high-quality creative services around the design crowd and the pan-office crowd.*
+
+## (2) Link to Project UI
+
+*A preview of the key interface of CloudSpeed shown below:*
+
+![CloudSpeed](https://ipfs.io/ipfs/QmQuAewRztFsztsELjDUtX6AMUwT78Bqr2XkUiFY51huU6 "CloudSpeed")
+
+*Project's github link: [CloudSpeed](https://github.com/deng/CloudSpeed)*
+
+
+## (3) What does your application/UI do?
+
+*Share files anytime, anywhere.*
+
+## (4) If your project is using a curated dataset, which one are you using?
+
+*Movie video scenery picture user application, science and technology literature education material and so on.*
```

Comment:
Can you reference a particular item in the datasets.md file or add any additions for your project via a separate PR?

Reply:
OK.

Reply:
OK